src/products/template.yaml

```diff
@@ -26,8 +26,10 @@ Resources:
       AttributeDefinitions:
         - AttributeName: name
           AttributeType: S
+        - AttributeName: sku   
+          AttributeType: S
       KeySchema:
-        - AttributeName: name
+        - AttributeName: sku
```

Comment:
I would suggest the opposite sku the primary, and name as sort, so we can query using sku